rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml

```diff
@@ -0,0 +1,2 @@
+:foreman_discovery:
```

Comment:
Yep.  bootdisk is a bad example, but salt has it: https://github.com/theforeman/hammer_cli_foreman_salt

(Means we can re-use it between RPMs and debs, and the maintainer can add new options in the example file.)